packages/kit/src/exports/index.js

```diff
@@ -72,3 +73,23 @@ export function text(body, init) {
 export function fail(status, data) {
 	return new ActionFailure(status, data);
 }
+
+/**
+ * Populate a route ID with params to produce relative path.
```

Comment:
Is it a relative path though? Bikeshedding, I know, but it's root relative, therefore could also be seen as absolute.
```suggestion
 * Populate a route ID with params to produce a relative path.
```

Reply:
```suggestion
 * Populate a route ID with params to resolve a pathname.
```